File: pingyougousellergoodsinterface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;


import com.pinyougou.pojo.TbBrand;
import entity.PageBean;

import java.util.List;
import java.util.Map;

public interface BrandService {


    public List<TbBrand> queryAll();


    public PageBean fillPage(int pageNum,int pageSize);

    public void add(TbBrand brand);

    public TbBrand queryOne(long id);

    public void update(TbBrand brand);

    public void delete(long[] ids);

    public PageBean fillPage(TbBrand brand,int pageNum,int pageSize);

    public List<Map> selectOptionList();

}

File: pingyougoushopweb/src/main/webapp/js/service/BrandService.js
app.service("brandService", function ($http) {
    this.queryAll = function (url) {
        return $http.get(url);
    }

    this.fillPage = function (page, size) {
        return $http.get('../brand/fillPage.do?page=' + page + '&size=' + size);
    }
    this.queryOne = function (id) {
        return $http.get('../brand/queryOne.do?id=' + id);
    }
    this.remove = function (ids) {
        return $http.get('../brand/delete.do?ids=' + ids);
    }
    this.add = function (entity) {
        return $http.post('../brand/add.do', entity);
    }
    this.update = function (entity) {
        return $http.post('../brand/update.do', entity);
    }
    this.search = function (page, size, domain) {
        return $http.post('../brand/search.do?page=' + page + '&size=' + size, domain);
    }

    this.selectOptionList=function () {
        return $http.get('../brand/selectOptionList.do');
    }


});